safeeyes/plugins/healthstats/plugin.py

```diff
@@ -55,7 +55,7 @@ def init(ctx, safeeyes_config, plugin_config):
             'total_resets': 0,
         }
 
-        session = context['session']['plugin'].get('healthstats', {}) | defaults
+        session = context['session']['plugin'].get('healthstats', {}) or defaults
```

Comment:
This is incorrect, I'm afraid - `a | b` will perform a union operation between the two dicts, preferring keys/values from `a` and falling back to keys/values from `b`.
However, `a or b` will pick either `a` or `b` in its entirety, depending on if `a` is empty or not.

So to keep the same behaviour, this would need to use `update()` on a copy, as the [PEP](https://peps.python.org/pep-0584/) recommends.